draft-ietf-scitt-architecture.md

```diff
@@ -326,6 +326,15 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
+{{fig-concept-relationship}} illustrates two Transparency Services and two Receipts as a single Signed Statement MAY be registered on more than one Transparency Service.
```

Comment:
Based on the result of #300, this text may be at the bottom of that merged text.
Or, if the text solves #268, this may make sense to move to the bottom of the diagram.